src/content/docs/git.md

```diff
@@ -37,9 +37,9 @@ The following table gives an overview about DataPLANT tools and services related
 
 Name | Type | Tasks on data sharing
 ----------------|-----------|------------------
-**[ArcCommander][KB-arccommander]** | Tool | **Collect, structure and share:** <ul><li>Add bibliographical metadata to your ARC</li><li>ARC version control and sharing via DataPLANT's DataHUB</li><li>Automated metadata referencing and version control as your ARC grows</li></ul>
-**[ARC][KB-ARC]**  <br> (Annotated Research Context) | Standard | <ul><li>ARCs are git repositories</li><li>Package data with metadata in a defined format</li></ul>
-**[DataHUB][DataHUB]** | Service | <ul><li>DataPLANT-customized GitLab instance</li><li>Infrastructure-as-code: on-premise solution</li><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
+**ArcCommander** | Tool | **Collect, structure and share:** <ul><li>Add bibliographical metadata to your ARC</li><li>ARC version control and sharing via DataPLANT's DataHUB</li><li>Automated metadata referencing and version control as your ARC grows</li></ul>
+**[ARC][KB-ARC]**  <br> (Annotated Research Context) | Standard | **Structure:** <ul><li>ARCs are git repositories</li><li>Package data with metadata in a defined format</li></ul>
+**[DataHUB](datahub.html)** | Service | **Share:** <ul><li>DataPLANT-customized GitLab instance</li><li>Infrastructure-as-code: on-premise solution</li><li>Federated system to share ARCs</li><li>Manage who can view or access your ARC</li></ul>
```

Comment:
Maybe add a cross reference here, so if a link gets changed we only need to change it in one place.

```
[KB-Datahub]: ./datahub.html
```

Reply:
Should have been fixed in [#77](https://github.com/nfdi4plants/nfdi4plants.github.io/pull/77/commits). Not sure why this one was closed...